components/org.wso2.carbon.metrics.impl/src/test/java/org/wso2/carbon/metrics/impl/MetricApiTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+package org.wso2.carbon.metrics.impl;
+
+import junit.framework.TestCase;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Assert;
+import org.wso2.carbon.metrics.common.MetricsConfiguration;
+import org.wso2.carbon.metrics.manager.*;
```

Comment:
Fixed as of #39. commit:[0402c09](https://github.com/wso2/carbon-metrics/blob/0402c09a877566689716337bba59c23d897bc07a/components/org.wso2.carbon.metrics.impl/src/test/java/org/wso2/carbon/metrics/impl/MetricApiTest.java)